lib/notifications.js

```diff
@@ -80,6 +216,16 @@ module.exports = ({ sqsConfiguration = {}, dbModels = {}, queueUrl }) => {
     } catch (err) {
       retryMessage(message, retry, err);
     }
+
+    // Then send the message to mq, only in the first attempt
+    if (retry) {
```

Comment:
I think sendMessage should only send SQS messages and separate it from MQ retry. Can we handle that in the service that invokes the functions (eg. platform-core)?